### PR TITLE
Add section on keeping blueprints up to date

### PR DESCRIPTION
**Pull request by axilleas**
## Proposed change

Add section on keeping blueprints up to date.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [x] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: https://github.com/home-assistant/home-assistant.io/issues/19766

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Comment by axilleas** [comment]
@frenck :wave: so this is a stripped down version of https://github.com/home-assistant/home-assistant.io/pull/20012, where I added only the blueprints update section. Cheers!

**Comment by frenck** [comment]
Not stale bot, thanks, awaiting our review and handling

File: source/_docs/automation/using_blueprints.markdown [hunk 1]
```diff
@@ -37,6 +37,27 @@ This will load the blueprint and show a preview in the import dialog. You can ch
...
 
+## Keeping blueprints up to date
+
+Blueprints created by the community may go through multiple revisions. Sometimes a user creates a blueprint,
+the community provides feedback, and new functionality is added.
+
+While there's no built-in functionality to update a blueprint you've already imported, you can manually edit
+its YAML content to keep it up to date:
+
+1. Navigate to the blueprints directory (`/blueprints/automation/`).
```
Comment (frenck) [inline source/_docs/automation/using_blueprints.markdown]: Super small details, but starting with `/` can be confused as starting from root. While you meant to say: relative to the configuration folder.

```suggestion
1. Navigate to the blueprints directory (`blueprints/automation/`).
```

File: source/_docs/automation/using_blueprints.markdown [hunk 2]
```diff
@@ -37,6 +37,27 @@ This will load the blueprint and show a preview in the import dialog. You can ch
...
+   The location of this diectory depends on the installation method and it's
+   similar to how you find [`configuration.yaml`](/docs/configuration/#editing-configurationyaml).
```
Comment (frenck) [inline source/_docs/automation/using_blueprints.markdown]:
```suggestion
   The location of this directory depends on the installation method and it's
   similar to how you find [`configuration.yaml`](/docs/configuration/#editing-configurationyaml).
```

**Review by frenck — CHANGES_REQUESTED** [review]
Looks good! 2 Tiny comments.

I do wonder, should it list something about the new Blueprint that might be incompatible with the previous version? And so it might need that you need to adjust them accordingly?

**Review by frenck — APPROVED** [review]
Thanks, @axilleas 👍

**Pull request merged**